CHANGES.md

```diff
@@ -53,6 +53,8 @@ Format of the entries must be.
 
 * Replaced dcos-metrics with Telegraf (DCOS_OSS-3714)
 
+* DCOS_METRICS_-prefixed labels are no longer included in metrics metadata by default, but are now configured to be (DCOS-43591)
```

Comment:
This and the entry below should be in the section above (1.12-dev), not here in 1.12.0. 

Maybe we could simplify it a little too, eg 'Made the prefix for metrics labels configurable'.